### PR TITLE
WIP (DO NOT MERGE): Initial backend wespire function

### PR DESCRIPTION
**Pull request by LinkMJB**
#### What does this PR do?
Adds initial wespire authentication and DOM+HTML download. Need to work on UI component still. Planning to add a slider/carousel to cycle through recent blog items.

#### Who is reviewing it?

@shaundmorris 

@gordocanchola

#### How should this be tested?
```
export WESPIRE_TOKEN=<remember_token>
make go
http://localhost:3000/tv
```

#### Checklist:
- [ ] Documentation Updated

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,9 +4,13 @@ FROM node:latest as build
 ARG SLACK_CHANNEL
 ARG SLACK_TOKEN
 ARG GITHUB_TOKEN
+ARG WESPIRE_TOKEN
```
Comment (shaundmorris) [inline Dockerfile:7]: Okay so since this runs on docker, we could just put a docker image on an EC2 on AWS to host this?

File: server/app.js [hunk 2]
```diff
@@ -17,6 +18,8 @@ const app = express();
 /* URL */
 const soaesb_url =
```
Comment (shaundmorris) [inline server/app.js:19]: ❔ So if we make this public we may want to remove/hide some of these that are hosted internally?

File: server/wespire.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                        style: style,
+                        date:  date,
+                        link:  link,
+                        text:  text,
+                    });
+                });
+            })
+
+      /*
+      blog.forEach((results) => {
```
Comment (shaundmorris) [inline server/wespire.js:72]: ❔  do you want to keep this?

Comment (gordocanchola) [reply server/wespire.js:72]: It's still a WIP so there is a lot that we'll end up cleaning up/getting rid of.

Comment (shaundmorris) [reply server/wespire.js:72]: I was just pretending to sound intelligent :-)

**Comment by shaundmorris** [comment]
Fantastic start @LinkMJB!

**Pull request closed without merging**